### PR TITLE
Wol ip address

### PR DESCRIPTION
**Pull request by smaugs**
Added an option to specify an IP Adress to wake up. Needed when waking up hosts in another subnet (no in the same broadcast domain).

### CODE REVIEW

File: wol/__init__.py [hunk 1]
```diff
@@ -27,10 +27,13 @@
 #        class_name = WakeOnLan
 #        class_path = plugins.wol
 #
-#  2. add to you items atribute  "wol_mac" with the mac for wake up
+#  2. add to you items atribute  "wol_mac" with the mac for wake up.
+#     there is also an option wol_ip when waking up hosts in differen
```
Comment (ohinckel) [inline wol/__init__.py]: Just a typo in different.

File: wol/README.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ Configuration
      type = bool
 
      wol_mac = 01:02:03:04:05:06
+     wol_ip = 1.2.3.4
```
Comment (ohinckel) [inline wol/README.md]: This line should start wih a '#' to indicate this is an optional argument.

**Comment by smaugs** [comment]
thx for review. done as requested.

**Comment by psilo909** [comment]
@ohinckel can we merge?

**Comment by ohinckel** [comment]
I think looks good now. Thanks for take care!

**Pull request merged**